IdentityCore/src/throttling/model/MSIDThrottlingModel429.h

```diff
@@ -25,7 +25,7 @@
 #import "MSIDThrottlingModelBase.h"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->